interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -388,6 +414,23 @@ function ViewModel() {
             self.isRestarting(1)
         }).always(self.setNextUpdate);
 
+        // this is a test of getting the entire queue
```

Comment:
and you are right totally not a test.  :)  i can change the logic to be purely based on the numbers from the queue itself.  instead of a second call to the queue api if you think the future of the feature is not going to need need knowledge of the entire queue before making a decision on where to go.